tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/ManifestFilters.scala

```diff
@@ -4,7 +4,7 @@ package org.scalajs.core.tools.jsdep
 object ManifestFilters {
 
   type ManifestFilter =
-    Traversable[JSDependencyManifest] => Traversable[JSDependencyManifest]
+    Iterable[JSDependencyManifest] => Iterable[JSDependencyManifest]
```

Comment:
That is not an acceptable change, because it breaks a public API.

You need to use a `type` alias in an `Compat` object instead.